releaseNote.md

```diff
@@ -13,6 +13,10 @@
 - Return exit code when MANUALLY_TRAP_SIG is exported (#2285)
 - Use results for uploading step summaries (#2301) with limited size (#2321)
 
+_Note: Actions Runner follows a progressive release policy, so the latest release might not be available to your enterprise, organization, or repository yet. 
```

Comment:
move to the top or here?

Reply:
I thought it might be distracting from the actual release changes if it was up top so I put it at the top of the repeat information. No strong preference if you prefer up top